pkg/profilecreator/profilecreator_test.go

```diff
@@ -498,3 +499,45 @@ var _ = Describe("PerformanceProfileCreator: Test Helper Function ensureSameTopo
 		})
 	})
 })
+
+var _ = Describe("PerformanceProfileCreator: Test Helper Function GetAdditionalKernelArgs", func() {
+	var powerMode string
+	// var args []string
+	Context("Ensure kernel args are populated correctly", func() {
+		It("Ensure kernel args are populated correctly in case of low-latency ", func() {
+			powerMode = "default"
+			kernelArgs := GetAdditionalKernelArgs(powerMode)
+			Expect(kernelArgs).To(BeEquivalentTo([]string{}))
+		})
+
+	})
+	Context("Ensure kernel args are populated correctly", func() {
+		It("Ensure kernel args are populated correctly in case of low-latency ", func() {
+			powerMode = "low-latency"
+			args := []string{"audit=0",
+				"mce=off",
+				"nmi_watchdog=0",
+			}
+			kernelArgs := GetAdditionalKernelArgs(powerMode)
+			sort.Strings(kernelArgs) // sort to avoid inequality due to difference in order
```

Comment:
good idea, but the `GetAdditionalKernelArgs` contract doesn't guarantee a sorted output, so we should sort both.
We can do that on a later PR.